source/deploying-with-kubernetes/kubernetes-upgrade.rst

```diff
@@ -0,0 +1,212 @@
+    <group name="local,syslog,sshd,">
+
+    <!--
+    Dec 10 01:02:02 host sshd[1234]: Failed none for root from 1.1.1.1 port 1066 ssh2
+    -->
+    <rule id="100001" level="5">
+        <if_sid>5716</if_sid>
+        <srcip>1.1.1.1</srcip>
+        <description>sshd: authentication failed from IP 1.1.1.1.</description>
+        <group>authentication_failed,pci_dss_10.2.4,pci_dss_10.2.5,</group>
+    </rule>
+
+    <rule id="100002" level="5">
+        <if_sid>5716</if_sid>
+        <srcip>2.1.1.1</srcip>
+        <description>sshd: authentication failed from IP 2.1.1.1.</description>
+        <group>authentication_failed,pci_dss_10.2.4,pci_dss_10.2.5,</group>
+    </rule>
+
+    <rule id="100003" level="7">
+        <if_sid>5716</if_sid>
+        <srcip>3.1.1.1</srcip>
+        <description>sshd: authentication failed from IP 3.1.1.1.</description>
+        <group>authentication_failed,pci_dss_10.2.4,pci_dss_10.2.5,</group>
+    </rule>
+    </group>
+    
+.. code-block:: xml
+
+    root@wazuh-manager-master-0:/# cat /etc/postfix/etc/rules/local_rules.xml
+    <!-- Local rules -->
+
+    <!-- Modify it at your will. -->
+
+    <!-- Example -->
+    <group name="local,syslog,sshd,">
+
+    <!--
+    Dec 10 01:02:02 host sshd[1234]: Failed none for root from 1.1.1.1 port 1066 ssh2
+    -->
+    <rule id="100001" level="5">
+        <if_sid>5716</if_sid>
+        <srcip>1.1.1.1</srcip>
+        <description>sshd: authentication failed from IP 1.1.1.1.</description>
+        <group>authentication_failed,pci_dss_10.2.4,pci_dss_10.2.5,</group>
+    </rule>
+
+    <rule id="100002" level="5">
+        <if_sid>5716</if_sid>
+        <srcip>2.1.1.1</srcip>
+        <description>sshd: authentication failed from IP 2.1.1.1.</description>
+        <group>authentication_failed,pci_dss_10.2.4,pci_dss_10.2.5,</group>
+    </rule>
+
+    <rule id="100003" level="7">
+        <if_sid>5716</if_sid>
+        <srcip>3.1.1.1</srcip>
+        <description>sshd: authentication failed from IP 3.1.1.1.</description>
+        <group>authentication_failed,pci_dss_10.2.4,pci_dss_10.2.5,</group>
+    </rule>
+
+    </group>
+
+At this point, if the pod was dropped or updated, Kubernetes would be in charge of creating a replica of it that would link to the volumes created and would maintain any changes referenced in the files and directories that we export to those volumes.
+
+Once explained the operation regarding the volumes, we proceed to update Wazuh in two simple steps.
+
+Change the image of the container
+---------------------------------
+
+The second step is to change the image of the pod in each file that deploys each node of the Wazuh cluster.
+
+These files are the *statefulSet* files:
+
+    - wazuh-master-sts.yaml
+    - wazuh-worker-0-sts.yaml
+    - wazuh-worker-1-sts.yaml
+
+For example we had this version before:
+
+.. code-block:: bash
+
+    containers:
+    - name: wazuh-manager
+      image: 'wazuh/wazuh:3.9.0_6.6.2'
+
+.. code-block:: bash
```

Comment:
It's not `bash` neither `console`, think about other syntax highlighters such as `yaml`